chain.go

```diff
@@ -193,6 +193,62 @@ func (cc *Conn) ListChains() ([]*Chain, error) {
 	return cc.ListChainsOfTableFamily(TableFamilyUnspecified)
 }
 
+func (cc *Conn) ListChain(table *Table, chain string) (*Chain, error) {
+	conn, closer, err := cc.netlinkConn()
+	if err != nil {
+		return nil, err
+	}
+	defer func() { _ = closer() }()
+
+	tableAttr := []netlink.Attribute{
+		{Type: unix.NFTA_TABLE_NAME, Data: []byte(table.Name + "\x00")},
+	}
+	flags := netlink.Request
+	msgs := []netlink.Message{
+		{
```

Comment:
The result of this first GETTABLE request seems to be discarded later? Could you add a comment explaining why two requests need to be sent instead of just the GETCHAIN request?

Reply:
Thanks for pointing this out. It seems that you were right. The `nft` CLI, which I used as reference, seems to send a message with type 2561 (GET_TABLE) before sending type 2564 (GET_CHAIN). I took a look at libnftnl source code and confirmed that a single GET_CHAIN request is enough: http://git.netfilter.org/libnftnl/tree/examples/nft-chain-get.c?id=be0bae0ad31b0adb506f96de083f52a2bd0d4fbf#n89